webpack.config.js

```diff
@@ -35,7 +35,8 @@ const config = module.exports = {
     ]
   },
   entry: {
-    'metamaps.bundle': './frontend/src/index.js'
+    'metamaps.bundle': './frontend/src/index.js',
+    'homepage': './frontend/src/homepage.js',
```

Comment:
The homepage is actually loading it's entirely own thing. The plan is to not include the main bundle and all its code on the homepage